recipes/mtsv/build.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash -e
+export HOME="/Users/distiller"
+export CC=$GCC
+export CARGO_TARGET_X86_64_UNKNOWN_LINUX_GNU_LINKER="${CC}"
+cd mtsv/ext
```

Comment:
Some comments in here explaining why these things need to be done would be good.

Reply:
Added comment